Artesian/_Services/Exceptions.py

```diff
@@ -0,0 +1,125 @@
+
+
+class ArtesianSdkException(Exception):
+    """
+        Base class for all `Artesian` errors.
+    """
+
+    def __init__(self, message: str) -> None:
+        """
+            Constructor.
+
+            Args:
+                message: the message describing the problem.
+        """
+        Exception.__init__(self, message)
+        self._message = message
+
+    @property
+    def message(self):
+        return self._message
+
+        
+class ArtesianSdkRemoteException(ArtesianSdkException):
+    """
+        Raised when the Artesian Service validation of an object failed.
```

Comment:
check documentation of all exceptions. This is a generic remote not a 'validation'